react-app/components/DropZoneFile/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useContext, memo } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFileImage } from '@fortawesome/free-regular-svg-icons';
+import { GalleryContext } from '@contexts/galleryContext';
+import { getItemLocalStorage, setItemLocalStorage } from '@utils/localStorage';
+import { sizeFileFormat } from '@utils/fileUtils';
+import './styles.css';
+
+const fileTypesAllowed = {
+  'image/jpeg': [],
+  'image/png': [],
+  'image/webp': [],
+  'image/heic': [],
+  'image/jfif': [],
+  'image/gif': []
+};
+
+const DropZoneFile = () => {
+  const { setFilesToUploadList } = useContext(GalleryContext);
+
+  const onDrop = useCallback((acceptedFiles) => {
```

Comment:
what is an `acceptedFiles`? what is `accepted` in this context?

Reply:
Only allows images -> .png, jpg....

Reply:
Then Allowed could be a more fitting word